test/auth.test.ts

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest';
+import { server } from '../src/index';
+import { app } from '../src/app';
+import prisma from '../src/utils/client';
+
+const req = request(app);
+interface IUserInfo {
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+    confirmPassword: string;
+}
+
+interface IRole {
+    id?: number;
+    name: string;
+    code: string;
+}
+
+const roleData: IRole = {
+    id: 5,
+    name: 'Guest',
+    code: 'roleCode',
+};
+
+const userData: IUserInfo = {
+    firstName: 'Afra',
+    lastName: 'Güngör',
+    email: 'afra.development@gmail.com',
+    password: 'Afra123!',
+    confirmPassword: 'Afra123!',
+};
+
+beforeAll(async () => {
+    await prisma.role.create({ data: roleData });
+});
+
+describe('Authentication tests', () => {
+    describe('Register route', () => {
+        it('should return 400 when required parameters are not provided', async () => {
+            await req
```

Comment:
Please have a look at the 3A of testing. [https://dev.to/coderjay06/the-three-a-s-of-unit-testing-b22](https://dev.to/coderjay06/the-three-a-s-of-unit-testing-b22)

You can modify your test suites accordingly for better readability.